blocks/leader-profiles/leader-profiles.js

```diff
@@ -78,6 +78,15 @@ export default function decorate(block) {
           icon.closest('a').target = '_blank';
           icon.closest('ul').classList.add('leader-profile-social-icons');
         });
+        if (block.matches('.block-option')) {
```

Comment:
I think there is a misunderstanding, the name of the block option should not be "block option", that's the name of the feature itself, please clarify with @DaphneKoren what the name should be

Reply:
Hi Yes, I think the name  "ION" should do for this block option.